### PR TITLE
Set/clear $VIRTUAL_ENV on activate/deactivate

### LINKED ISSUE #26: How to update Powerline when activating?
I tend to use virtualenvwrapper in when in my shell, and your plugin when in vim. I've done a lot of stuff in the background with zsh to try and keep my env-vars updated when changing directories and make sure that your plugin looks at the same virtualenv as virtualenvwrapper.

So that said, when I activate a venv in zsh, then enter vim, powerline shows me the name of my virtualenv in a little indicator. The code it uses is [here](https://github.com/Lokaltog/powerline/blob/develop/powerline/segments/common.py#L828). And I have it working correctly for that particular situation. When I enter vim however without activating a venv first, and use your plugin to activate the same venv, the python path is modified correctly and works just fine, but powerline hasn't been notified that the venv was activated and my powerline indicator doesn't show up. 

Any insight on how to modify your plugin to write the appropriate env-vars and announce to powerline to update that particular segment?

### PR DESCRIPTION
**Pull request by brianclements**
Set this so that other plugins that are configured to look for it
(Powerline) can now do that and behave accordingly.

Powerline was patched here:
https://github.com/Lokaltog/powerline/pull/896

To allow this functionality from it's end.

This patch also fixes #26.

### CODE REVIEW

File: autoload/virtualenv.vim [hunk 1]
```diff
@@ -55,6 +56,7 @@ EOF
     endif
     unlet! g:virtualenv_name
     unlet! g:virtualenv_path
+    let $VIRTUAL_ENV = '' " can't delete parent variables
```
Comment (jmcantrell) [inline autoload/virtualenv.vim]: Would `unlet` not work here, or is that the meaning of the comment?

File: doc/virtualenv.txt [hunk 2]
```diff
@@ -51,4 +51,8 @@ To use the statusline flag, this must appear in your |'statusline'| setting: >
 <
 The content is derived from the |g:virtualenv_stl_format| variable.
 
+Powerline users can see their statuslines too. After configuring powerline
+according to it's documentation to support a virtualenv segment, Powerline
```
Comment (jmcantrell) [inline doc/virtualenv.txt]: s/it's/its/

Sorry, I know this is minor.

File: autoload/virtualenv.vim (hunk 1 above)
Comment (brianclements) [inline autoload/virtualenv.vim]: correct, if vim starts with VIRTUAL_ENV already set from the environment, then it won't be able to delete the var, only empty it as I did here.

File: doc/virtualenv.txt (hunk 2 above)
Comment (brianclements) [inline doc/virtualenv.txt]: Its weird, I usually get that right.....

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+/doc/tags
```
Comment (jmcantrell) [inline .gitignore]: Good catch, thanks!

**Pull request merged**